lib/deck.rb

```diff
@@ -0,0 +1,22 @@
+class Deck
+    attr_reader :cards
+
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def count
+    cards.count
```

Comment:
Instead of calling `cards.count`, call to the instance variable `@cards.count`. When you call to `cards` it is actually calling to the attr_Reader on line 2, which then returns the instance variable `@cards`. To optimize your code, cut out the middle man and call directly to `@cards`